others/_posts/2017-09-18-sweat-and-toil.md

```diff
@@ -773,7 +773,7 @@ Forced Labor, and Human Trafficking Around the World. For more information, see
           Country Name
         </td>
       </tr>
-      <tr>
+      <!--- <tr>
```

Comment:
remove the code if not needed.

Reply:
removed unwanted code and modified some text in the description part. The branch CAS-170 has been updated with changes.